api/security/policy.go

```diff
@@ -0,0 +1,85 @@
+package security
+
+import (
+	"net/http"
+
+	"github.com/gobwas/glob"
+)
+
+// HasOne returns a gin handler that checks the request against the given role
+func (policy RolePolicy) HasOne(roles ...Role) bool {
+	for _, role := range roles {
+		if policy[role] {
+			return true
+		}
+	}
+	return false
+}
+
+// HasAll returns a gin handler that checks the request against the given role
```

Comment:
Unused. Is it here for the sake of completeness ?

Reply:
And to ease testability